web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -121,6 +127,22 @@ public ResponseEntity runValidator(
     }
   }
 
+  @GetMapping("/version")
+  public ResponseEntity currentVersion() {
+    // https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app/version
+    StringBuilder stringBuilder = new StringBuilder();
+    VersionResolver checker = new VersionResolver();
+    VersionInfo versionInfo = checker.getVersionInfoWithTimeout(TIMEOUT);
+    stringBuilder.append(versionInfo.currentVersion());
```

Comment:
The current version is of class `Optional`. Let's return the string if available. If not available, we can throw an exception with an explanatory message; this should not happen with a properly packaged jar.